samples/angular-todo/src/app/features/journey/identityProvider/identity-provider.component.html

```diff
@@ -0,0 +1,14 @@
+<div *ngFor="let provider of identityProviders">
+  <button
+    [ngClass]="{
+      'google-login': getDisplayName(provider) === idpNames.Google,
+      'apple-login': getDisplayName(provider) === idpNames.Apple,
+      'facebook-login': getDisplayName(provider) === idpNames.Facebook
+    }"
+    (click)="onSetProvider(provider.provider)"
+  >
+    <app-google-icon *ngIf="getDisplayName(provider) === idpNames.Google"></app-google-icon>
+    <app-apple-icon *ngIf="getDisplayName(provider) === idpNames.Apple"></app-apple-icon>
+    Sign in with {{ getDisplayName(provider) }}
```

Comment:
Does Facebook default to this generic string template here, without an icon?

Reply:
it has no icon before text
<img width="478" alt="Screenshot 2023-08-08 at 15 18 26" src="https://github.com/ForgeRock/forgerock-javascript-sdk/assets/18680830/bb5173df-5acc-4933-a85b-387d814172ee">

Reply:
Thanks for the screenshot. Can you remove the default borders around the buttons, so they follow the same style convention as the other buttons in the app? Only the Google button needs a border, since the background is white, so let's use the same style is the "outline button" that should already be defined in the Bootstrap library. The outline button should have the same style as the other form elements.